src/controllers/orgContentController.js

```diff
@@ -1,44 +1,47 @@
+/* eslint-disable no-undef */
 const path = require('path');
 const fs = require('fs');
 const { renderTemplate, renderTemplateFromAPI } = require('../utils/util');
-const config = require('../config/config');
+const config = require(process.cwd() + '/config.json');
+const constants = require('../utils/constants');
+const adminDao = require('../dao/admin');
 
-
-filePrefix = '../../../../src/'
+const filePrefix = config.pathToContent;
 
 
 const loadOrganizationContent = async (req, res) => {
 
     let html = "";
-    if (config.mode == 'single' || config.mode == 'design') {
+    if (config.mode === constants.DEV_MODE) {
         html = await loadOrgContentFromFile(req, res)
     } else {
         html = await loadOrgContentFromAPI(req, res)
     }
     res.send(html);
 }
 
-const loadOrgContentFromFile = async (req, res) => {
+const loadOrgContentFromFile = async () => {
 
     //TODO fetch from DB
-    const mockProfileDataPath = path.join(__dirname, filePrefix + '../mock', '/userProfiles.json');
+    const mockProfileDataPath = path.join(process.cwd(), filePrefix + '../mock', '/userProfiles.json');
     const mockProfileData = JSON.parse(fs.readFileSync(mockProfileDataPath, 'utf-8'));
 
-    let baseURL = "http://localhost:" + config.port;
-    if (config.mode == 'single')
-        baseURL = req.params.orgName
     let templateContent = {
         userProfiles: mockProfileData,
-        baseUrl: baseURL
+        baseUrl: constants.BASE_URL + config.port
     };
     return renderTemplate(filePrefix + 'pages/home/page.hbs', filePrefix + 'layout/main.hbs', templateContent)
 }
 
-const loadOrgContentFromAPI = async (req, res) => {
+const loadOrgContentFromAPI = async (req) => {
 
     let templateContent = {}
```

Comment:
Instead of assigning these variables can set the value directly to the `renderTemplateFromAPI` function

Reply:
changed